pegasus/sites.v3/code.org/views/page_not_available.haml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
You had to make this page from scratch?

Reply:
oops, removed the unnecessary lines at the top

Reply:
Hah didn't even notice that, I just meant we don't have a page not available view already (looks like maybe not browsing through pegasus views...)

Reply:
oh yeah we have a page not found but we want something more custom that points people to support if they think they need access